src/vis/vis.js

```diff
@@ -427,14 +437,17 @@ var Vis = View.extend({
       $(window).bind('resize', this._onResize);
     }
 
+    // Create a concrete MapView subclass from map model configuration.
     var div = $('<div>').css({
       position: 'relative',
       width: '100%',
       height: '100%'
     });
-
-    this.container = div;
-
+    this.$el.append(div);
+    //
+    // TODO - Is this really necessary? I have tested in the examples and works
+    // fine without it.
+    //
```

Comment:
Not sure about this. I'd test it a bit more and if you think we can remove it, I'd go ahead and remove it. You can also ask whoever did it in the first place, but I let's figure it out now instead of adding another TODO.